src/qibocal/protocols/characterization/ramsey.py

```diff
@@ -439,12 +402,23 @@ def fitting(x: list, y: list) -> list:
             [0, 0, 0, -np.pi, 0],
             [1, 1, np.inf, np.pi, np.inf],
         ),
-    )[0]
+        sigma=err,
+    )
     popt = [
-        (y_max - y_min) * popt[0] + y_min,
-        (y_max - y_min) * popt[1] * np.exp(x_min * popt[4] / (x_max - x_min)),
-        popt[2] / (x_max - x_min),
-        popt[3] - x_min * popt[2] / (x_max - x_min),
-        popt[4] / (x_max - x_min),
+        delta_y * popt[0] + y_min,
+        delta_y * popt[1] * np.exp(x_min * popt[4] / delta_x),
+        popt[2] / delta_x,
+        popt[3] - x_min * popt[2] / delta_x,
+        popt[4] / delta_x,
+    ]
+    perr = np.sqrt(np.diag(perr))
+    perr = [
+        delta_y * perr[0],
+        delta_y
+        * np.exp(-x_min * popt[4] / delta_x)
```

Comment:
```suggestion
        * np.exp(x_min * popt[4] / delta_x)
```
If I did the error propagation correctly this might be a + sign